chore: OPTIC-1778: Unify and prettify icons

yyassi-heartex opened the pull request:
we have icons across the different libraries that make up LabelStudio. this PR unifies them to remove duplicates + allows us to standardize them. will also run by design to make sure we update icons to a more modern look

yyassi-heartex committed:
fix: OPTIC-1748: [Timebox] UI polish

yyassi-heartex committed:
changing Delete back to Remove

yyassi-heartex committed:
standardizing utils/dom

yyassi-heartex committed:
fixing an issue that affects columns on datamanger table

yyassi-heartex committed:
lint cleanup

yyassi-heartex committed:
more cleanup

yyassi-heartex committed:
tweaking tooltip top-right positioning + view toggle switch to icons + comment item tooltip fixes

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1748/ui-improvements

yyassi-heartex committed:
lint cleanup

yyassi-heartex committed:
unifying Userpic

yyassi-heartex committed:
classname fix for Userpic

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1748/ui-improvements

yyassi-heartex committed:
adding test ids and updating how we check for toggle

yyassi-heartex committed:
lint cleanup

yyassi-heartex committed:
fixing column check

yyassi-heartex committed:
lint cleanup

yyassi-heartex committed:
fixed unit test + reduced padding on datamanger view toggle buttons

yyassi-heartex committed:
lint + test fixes

yyassi-heartex committed:
test fixes

yyassi-heartex committed:
addressing issues around datamanager columns checkboxes

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1748/ui-improvements

yyassi-heartex committed:
addressing failing checks

yyassi-heartex committed:
fixing userDisplayName

yyassi-heartex committed:
lint cleanup

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1748/ui-improvements

bmartel committed:
fixing AnnotationsCarousel tests

yyassi-heartex committed:
cleaning up a situation where its possible for user to be null when we try to spread its values

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1748/ui-improvements

yyassi-heartex committed:
chore: OPTIC-1778: Unify and prettify icons

yyassi-heartex committed:
slight tweaks

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-optic-1748/ui-improvements'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13697081739

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13697081739

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13697520796

yyassi-heartex committed:
test and lint cleanup

yyassi-heartex committed:
Merge branch 'fb-optic-1748/ui-improvements' of github.com:heartexlabs/label-studio into fb-optic-1748/ui-improvements

yyassi-heartex committed:
fixing an issue that makes it impossible for a user to update the fields

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1748/ui-improvements

yyassi-heartex committed:
Revert "Merge remote-tracking branch 'origin/develop' into fb-optic-1748/ui-improvements"

This reverts commit 291f6e37083861000c5d3bbb1ff2e73e7119a2ce, reversing
changes made to 1c53cd9462bceb87ee0d9a499be0ec0b1410dce1.

yyassi-heartex committed:
Reapply "Merge remote-tracking branch 'origin/develop' into fb-optic-1748/ui-improvements"

This reverts commit 233340c6576e1af660ec6386eb45c2ac65e6d144.

yyassi-heartex committed:
fixing merge

yyassi-heartex committed:
cleaning up imports

yyassi-heartex committed:
Merge branch 'fb-optic-1748/ui-improvements' into fb-optic-1778/icons

yyassi-heartex committed:
addressing a different in icon usage

yyassi-heartex committed:
icon cleanup

yyassi-heartex committed:
import upload uptton sizing

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13716495646

yyassi-heartex committed:
Update web/libs/core/src/pages/AccountSettings/sections/PersonalInfo.tsx

yyassi-heartex committed:
Update web/libs/ui/src/lib/Userpic/Userpic.tsx

yyassi-heartex committed:
Update web/libs/ui/src/lib/Userpic/Userpic.tsx

yyassi-heartex committed:
unfiying icons

yyassi-heartex committed:
fixing icon

yyassi-heartex committed:
Merge branch 'develop' into fb-optic-1748/ui-improvements

yyassi-heartex committed:
cleaned out unused files + fixed tooltip breakage due to quick toggle + added intent delay

yyassi-heartex committed:
lint cleanup

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13763179195

yyassi-heartex committed:
we dont need this icon here

yyassi-heartex committed:
Merge branch 'fb-optic-1748/ui-improvements' into fb-optic-1778/icons

yyassi-heartex committed:
Merge branch 'develop' into fb-optic-1778/icons

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1778/icons

yyassi-heartex committed:
icon updates

yyassi-heartex committed:
lint cleanup

yyassi-heartex committed:
icon cleanup

yyassi-heartex committed:
react-icons no longer used + cleanup

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1778/icons

yyassi-heartex committed:
fixing icon import

yyassi-heartex committed:
Merge branch 'develop' into fb-optic-1778/icons

yyassi-heartex committed:
fixing e2e + integration tests

yyassi-heartex commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13796443131)
> Successfully merged:  19 files changed, 258 insertions(+), 67 deletions(-)

yyassi-heartex committed:
Merge branch 'develop' into 'fb-optic-1778/icons'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13796443131

yyassi-heartex committed:
slight tweaks

yyassi-heartex committed:
Merge branch 'fb-optic-1778/icons' of github.com:heartexlabs/label-studio into fb-optic-1778/icons

yyassi-heartex committed:
fixing an import

yyassi-heartex committed:
lint cleanup

yyassi-heartex committed:
unifiying LSE icons into ui library

yyassi-heartex committed:
adjusting Compare all annotations icon size

yyassi-heartex committed:
bigger button looks better

luarmr commented on web/libs/datamanager/src/components/Common/FiltersPane.jsx:
```diff
@@ -1,7 +1,6 @@
 import { inject, observer } from "mobx-react";
 import React, { useEffect, useRef } from "react";
-import { FaAngleDown, FaChevronDown } from "react-icons/fa";
-import { FF_LOPS_E_10, isFF } from "../../utils/feature-flags";
```
❤️

nass600 reviewed (commented):
My only comment is regarding icon naming and folder organization:
I am not sure if the naming follows this [figma](https://www.figma.com/design/y1mgRgbevFr3MRSz8qhklS/Label-Studio-Design-System?node-id=66-2529&t=KdMzbOLqzby3hOrd-0) or there are any other source
Regarding the structure with assets  I see some icons in categorized folders and other that aren't. Is it following any reasoning?

bmartel commented:
> My only comment is regarding icon naming and folder organization: I am not sure if the naming follows this [figma](https://www.figma.com/design/y1mgRgbevFr3MRSz8qhklS/Label-Studio-Design-System?node-id=66-2529&t=KdMzbOLqzby3hOrd-0) or there are any other source Regarding the structure with assets I see some icons in categorized folders and other that aren't. Is it following any reasoning?

@nass600 
The only reasoning here is it retained the folder structure of the existing icons when moved over to the UI lib, to cut down on amounts of changes required as the focus was on unification most of all in this ticket. I think it is worth doing though, which we have follow up tickets to address. In the follow up I want to have two things be true:

1. Standardized icons (organize and name them according to figma, also drop any which should not exist/tweak etc.)
2. All icons are available in Storybook

bmartel committed:
fix top bar icon sizing to match design intent

bmartel committed:
comment form icon styles

bmartel committed:
don't increase weight of icons by forcing a stroke to paths of button icons

bmartel committed:
fix eye icon sizing in relations and regions panel

bmartel committed:
linting

bmartel committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1778/icons

bmartel committed:
fix slider icon import

bmartel committed:
add new icons to replace MD Icons in app

bmartel committed:
added sync disabled icon

yyassi-heartex committed:
updating bell.svg

yyassi-heartex committed:
Merge branch 'fb-optic-1778/icons' of github.com:heartexlabs/label-studio into fb-optic-1778/icons

bmartel committed:
remove react-icons

bmartel committed:
Merge branch 'fb-optic-1778/icons' of github.com:HumanSignal/label-studio into fb-optic-1778/icons

AndrejOros commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13920015044)

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-optic-1778/icons'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13920015044

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13920015044

yyassi-heartex committed:
now using new file upload.svg

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1778/icons

yyassi-heartex committed:
resolving merge issue + restoring refresh icon

yyassi-heartex committed:
lint cleanup

AndrejOros commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13943734464)

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-optic-1778/icons'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13943734464

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13943734464

yyassi-heartex commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13947345721)
> Successfully merged: Already up to date.

yyassi-heartex committed:
updating modal close icon size

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1778/icons

yyassi-heartex committed:
lint cleanup

AndrejOros commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13965639081)

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-optic-1778/icons'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13965639081

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13965639081

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1778/icons

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1778/icons

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-1778/icons

yyassi-heartex committed:
splitting taxonomy test into 4 files to improve test stability

yyassi-heartex committed:
lint cleanup
